lib/src/utils.dart

```diff
@@ -30,6 +29,18 @@ import 'package:yaml/yaml.dart';
 
 import 'info.dart';
 
+/// The map of os to list of architectures for packaging.
+Map<String, List<String>> pkgOsArch = {
```

Comment:
Since this map isn't exported from the package, it doesn't need a `pkg` prefix. I'd call it something like `osToArchs`.